src/components/schedule-visualizer/robot-trajectories-overlay.tsx

```diff
@@ -26,6 +28,26 @@ export default function RobotTrajectoriesOverlay(
   // FIXME: hardcode for now, as the source of the footprint is expected to change.
   const footprint = 0.5;
 
+  useEffect(() => {
+    function getConflictRobotsName(): string[] {
+      let robotNames: string[] = [];
+      conflicts.forEach(conflictId => {
+        const robotName = DefaultTrajectoryManager.getRobotNameFromPathId(conflictId, trajs);
+
+        robotName && robotNames.push(robotName);
+      });
+      return robotNames;
+    }
+
+    if (conflicts.length !== 0) {
+      notificationDispatch &&
+        notificationDispatch({
+          message: `Trajectory conflict between ${getConflictRobotsName()}`,
```

Comment:
If I understand this correctly, it prints robots in all conflicting trajectories. For example if robot A conflicts with robot B and robot C conflicts with robot D, it would print A, B, C, D conflicting when there is actually 2 separate conflicts going on.

Reply:
Yeah, that's right. I'll remove the word between. 

We can just mention the conflicting trajectories for now. Unless I check the collision points by coordinates, but we discard that option [here](https://github.com/osrf/romi-dashboard/pull/40#issuecomment-649438833).

Let me know what you think about it.

Reply:
Thats a good point, the current conflict data can't differentiate between multiple separate conflicts. @Yadunund Is the proposal possible to implement in the trajectory server?